iap_watcher/startup_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+# Initial key fetch and cron job setup
```

Comment:
Please split this into two different code blocks, one for fetching the keys, and one for doing the cron gymnastics, and elaborate on why the cron is needed.

Reply:
Done. Also decided to fetch them every 2 hours, instead of every hour (when I reviewed the nginx module code, I saw that it's staleness time was twice what I remembered).